components/interface/VFBTermInfo/VFBTermInfo.js

```diff
@@ -510,6 +548,7 @@ export default class VFBTermInfoWidget extends React.Component {
       if (nodePresent === false) {
         this.data.unshift(data);
       }
+      this.refs.termInfoRef.setGraphsLinks(data);
```

Comment:
As discussed, this call can be moved inside setData since the data in input are the same